Sources/SafeDITool/SafeDITool.swift

```diff
@@ -55,13 +57,14 @@ struct SafeDITool: AsyncParsableCommand {
             parsedModule(loadSwiftFiles())
         )
 
+        let generator = try DependencyTreeGenerator(
+            importStatements: dependentModuleInfo.flatMap(\.imports) + additionalImportedModules.map { ImportStatement(moduleName: $0) } + module.imports,
+            typeDescriptionToFulfillingInstantiableMap: resolveSafeDIFulfilledTypes(
+                instantiables: dependentModuleInfo.flatMap(\.instantiables) + module.instantiables
+            )
+        )
```

Comment:
not entirely ideal to have to enter the `func resolveSafeDIFulfilledTypes(...) throws`  call if we don't need it... but also the throwing this method does is reasonably important so it's not a big deal